rtic-time/Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 name = "rtic-time"
-version = "1.3.0"
+version = "2.0.0"
 
 edition = "2021"
 authors = [
-  "The Real-Time Interrupt-driven Concurrency developers",
-  "Emil Fresk <emil.fresk@gmail.com>",
-  "Henrik Tjäder <henrik@tjaders.com>",
-  "Jorge Aparicio <jorge@japaric.io>",
-  "Per Lindgren <per.lindgren@ltu.se>",
+    "The Real-Time Interrupt-driven Concurrency developers",
+    "Emil Fresk <emil.fresk@gmail.com>",
+    "Henrik Tjäder <henrik@tjaders.com>",
+    "Jorge Aparicio <jorge@japaric.io>",
+    "Per Lindgren <per.lindgren@ltu.se>",
 ]
 categories = ["concurrency", "embedded", "no-std", "asynchronous"]
 description = "rtic-time lib TODO"
```

Comment:
Now seems a good time to fix this TODO :)

Reply:
Lol :D
Not entirely sure what to write :D

Reply:
Done